### DISCUSSION TITLE
Cannot refactor ... that is defined outside of an open project.

### DISCUSSION THREAD

**Discussion by mauricioelliff** [apache/netbeans#6166] [Q&A]
Hi. When I try to move a file using menu Refactor/Move, it shows me " Cannot refactor ... that is defined outside of an open project."
I don't know what could I do.
I try with restart the IDE, but don't work.
The file, of course is under my project.
I use CVS.

This is my Help/About: 
Product Version: Apache NetBeans IDE 17
Java: 11.0.18; OpenJDK 64-Bit Server VM 11.0.18+10-post-Debian-1deb11u1
Runtime: OpenJDK Runtime Environment 11.0.18+10-post-Debian-1deb11u1
System: Linux version 5.10.0-21-amd64 running on amd64; UTF-8; es_AR (nb)
Proyect properties:
Encoding: UTF-8
PHP: 7.3

**Comment by asbachb** [comment]
@mauricioelliff Can you provide a sample project for that behaviour? If there's nothing wrong with the project setup it sounds more like a bug to me.
